prime-router/src/main/kotlin/azure/ReportFunction.kt

```diff
@@ -135,23 +135,29 @@ class ReportFunction : Logging {
         val workflowEngine = WorkflowEngine()
         val actionHistory = ActionHistory(TaskAction.receive, context)
         var report: Report? = null
+        val verboseResponse = StringBuilder()
```

Comment:
I do not see the need for this stringBuilder.  Can be just a string.

Reply:
I think this was future feature building. There's probably a plan to do extra building of information into the dataset.